core/test/element_unary_operations_test.cpp

```diff
@@ -280,3 +280,51 @@ TYPED_TEST(ElementNegativeInfToNumTest, value_and_variance_out) {
   targetted_replacement_out_arg_test(element::negative_inf_to_num_out_arg, out,
                                      replaceable, nonreplaceable, replacement);
 }
+
+TEST(ElementReciprocalTest, unit) {
+  const units::Unit one_over_m(units::dimensionless / units::m);
+  EXPECT_EQ(element::reciprocal(one_over_m), units::m);
+}
+
+// Question here: should we use EXPECT_EQ, or rather EXPECT_FLOAT_EQ?
```

Comment:
Since we have a way of computing the result I would keep using `EXPECT_EQ` as you did. You are probably right that that might eventually lead to a failing test, e.g., if SIMD instructions for fast reciprocal started to be used, but we can worry about that once it happens (if ever).